### PR TITLE
Fix logger member not correct、some javadoc  problem

### PR DESCRIPTION
**Pull request by Shoothzj**
Fix java doc problem

Descriptions of the changes in this PR:
1. Some log in slf4j, the pattern number and the member log not equal
2. Some javadoc grammar problem

### CODE REVIEW

File: bookkeeper-server/src/main/java/org/apache/bookkeeper/client/RackawareEnsemblePlacementPolicyImpl.java [hunk 1]
```diff
@@ -269,9 +269,9 @@ public RackawareEnsemblePlacementPolicyImpl initialize(ClientConfiguration conf,
                 }
             } catch (RuntimeException re) {
                 if (!conf.getEnforceMinNumRacksPerWriteQuorum()) {
-                    LOG.error("Failed to initialize DNS Resolver {}, used default subnet resolver : {}",
+                    LOG.error("Failed to initialize DNS Resolver {}, used default subnet resolver : {} {}",
```
Comment (dlg99) [inline bookkeeper-server/src/main/java/org/apache/bookkeeper/client/RackawareEnsemblePlacementPolicyImpl.java:272]: i'd just flip the order of `re` and  `re.getMessage()` on the next line and leave an exception without matching `{}` to let logger print the stack. Or remove `re.getMessage` altogether with both `{}` at the end

**Review by dlg99 — COMMENTED** [review]
LGTM, one nit to fix

File: bookkeeper-server/src/main/java/org/apache/bookkeeper/client/RackawareEnsemblePlacementPolicyImpl.java (hunk 1 above)
Comment (Shoothzj) [reply bookkeeper-server/src/main/java/org/apache/bookkeeper/client/RackawareEnsemblePlacementPolicyImpl.java:272]: I remeber only one exception args can print without matching `{}`, I remove the re.getMessage.

**Comment by Shoothzj** [comment]
@eolivelli PTAL, thanks

**Review by merlimat — APPROVED** [review]

**Pull request merged**